code/modules/chemistry/tools/robospray.dm

```diff
@@ -0,0 +1,84 @@
+/obj/item/robospray
+	name = "cybernetic hypospray"
+	desc = "An automated injector for cyborgs."
+	icon = 'icons/obj/chemical.dmi'
+	inhand_image_icon = 'icons/mob/inhand/hand_medical.dmi'
+	item_state = "syringe_0"
+	icon_state = "hypo0"
+	var/inj_amount = 5
+	var/cooldown = 0
+	var/picker = 1
+	var/sound/sound_inject = 'sound/items/hypo.ogg'
+	var/botreagents = list(
+		"epinephrine" = 25,
+		"charcoal" = 25,
+		"anti_rad" = 25,
+		"saline" = 25,
+		"salicylic_acid" = 25,
+		"salbutamol" = 25,
+		"antihistamine" = 25,
+		"mannitol" = 25,
+		"proconvertin" = 25,
+		"atropine" = 25
+	)
```

Comment:
I see your point, but as borgs cost more resources, require a player to operate, and are generally meant to (at least looking at the engineering borg) work at the same level as everyone else in their department, I feel allowing them to have chems that are not at all rare might be the right choice. However, I have reduced the chems they get a bit by cutting proconvertin and atropine. What's your take on that?

Reply:
Hm. I'd still cut the antihistamine, one of epi/mannitol, and probably anti_rad and salicylic as well, to keep them down to ~1 chem to treat any given thing, instead of 2

Reply:
Hm, antihistamine I can see. mannitol is a bit out there for me, as brain damage is a really common killer. sacid can go, though, but radmeds should stay. Thoughts? That'd make the list be epinephrine, salbumatol, mannitol, saline, charcoal, and potassium-iodine.